price_feeder/tasks.py

```diff
@@ -360,12 +360,25 @@ def post_price(self, price_no_precision, info_contracts, task=None, global_manag
 
         # Multiply factor of the using gas price
         calculated_gas_price = using_gas_price * decimal.Decimal(self.options['gas_price_multiply_factor'])
-
+        # is a price re post
+        if 'receipt' in pending_tx_receipt and re_post:
+            if 'last_used_nonce' in global_manager:
```

Comment:
Is this is false, like for example when the job as just started, shouldn't it be retrieved by:
`last_used_nonce = web3.eth.getTransactionCount(account_address)`